packages/p2p/src/components/modal-manager/modal-manager-context-provider.tsx

```diff
@@ -78,8 +120,11 @@ const ModalManagerContextProvider = props => {
      * - **should_save_form_history**: `false` by default. If set to `true`, form values in modals that has a form with `ModalForm` component
      * will be saved when the modal is hidden and restored when modal is shown again.
      */
-    const hideModal = (options = {}) => {
-        const { should_save_form_history = false, should_hide_all_modals = false } = options;
+    const hideModal = (options?: THideModalOptions) => {
+        // eslint-disable-next-line no-param-reassign
+        if (!options) options = { should_save_form_history: false, should_hide_all_modals: false };
+
+        const { should_save_form_history, should_hide_all_modals } = options;
```

Comment:
idk if its better, we can remove eslint comment with this

```suggestion
        const default_options = { should_save_form_history: false, should_hide_all_modals: false };

        const { should_save_form_history, should_hide_all_modals } = options || default_options;
```